frontends/halo2/src/driver.rs

```diff
@@ -81,10 +86,10 @@ impl Driver {
     }
 
     /// Creates the IR context for the synthesized circuit.
-    fn get_or_create_ir_ctx<'drv, F: PrimeField>(
-        &'drv mut self,
-        syn: &CircuitSynthesis<F>,
-    ) -> &'drv IRCtx {
+    fn get_or_create_ir_ctx<'drv, F>(&'drv mut self, syn: &CircuitSynthesis<F>) -> &'drv IRCtx
```

Comment:
What is this `IRCtx`? I know this is mainly a style change but it got me wondering.